esbuild.js

```diff
@@ -1,14 +1,16 @@
 const esbuild = require('esbuild');
 const glob = require('tiny-glob');
 (async () => {
-    const tsxPaths = await glob('packages/components/**/*.tsx');
-    const componentPaths = tsxPaths.filter(filePath => !filePath.includes('.spec.tsx'));
-    esbuild.build({
```

Comment:
Why don't we filter the tests out anymore?

Reply:
now we transpile the register.ts files, however I have changed it to make it clearer.